docker-compose.test.yaml

```diff
@@ -0,0 +1,11 @@
+version: '3.4'
+services:
+  mine-support-payment-service:
+    volumes: 
+    - node_modules:/usr/src/app/node_modules
+    - .:/usr/src/app
+    - ./test-output:/usr/src/app/test-output
+    command: ./wait-for mine-support-postgres-payments:5432 -- ./wait-for amq:8161 -- ./migrate-test

```

Comment:
this test-compose fails when I run scripts/test -the second `wait-for` can't be found.
Looks like chaining the `wait-for` commands doesn't work, the multi stage compose where the migration is run could be used instead.

Also, when the tests do run the command never exits as an async operation has not finished, perhaps a DB connection is still open?

If the leak can't be detected Jest can be forced to exit with the '--forceExit' switch